lib/network/default.nix

```diff
@@ -0,0 +1,107 @@
+{ lib }:
+{
+  ipv6 = {
```

Comment:
for ergonomics it would be nicer to just use v6 and v4 but I don't have hard feelings here.

Reply:
Personally, I think the "ip" prefix gives more clarity.
`network.v6.func` looks weird.

Reply:
I agree, proper lingo everywhere is what we should aim for.